src/main/java/seedu/address/model/VersionedResumeBook.java

```diff
@@ -94,31 +94,35 @@ public boolean equals(Object other) {
 
         // state check
         return super.equals(otherVersionedResumeBook)
-                && resumeBookStateList.equals(otherVersionedResumeBook.resumeBookStateList)
+                //&& resumeBookStateList.equals(otherVersionedResumeBook.resumeBookStateList)
                 && currentStatePointer == otherVersionedResumeBook.currentStatePointer;
     }
 
     public ReadOnlyResumeBook getStatelessResumeBook() {
-        this.currentStatePointer = 0;
-        this.resumeBookStateList.clear();
-        return this;
+        ResumeBook newResumeBook = new ResumeBook();
```

Comment:
Nice catch, I reckon this is the 3 lines since the old way of doing things would destroy the resumeBookStateList and current state pointer? My bad dude, I think I was the one that wrote this, didn't have the foresight to think about when we're gonna write tests for undo and redo we'll still need the pointer and stateList :O

Reply:
:(